src/App.js

```diff
@@ -0,0 +1,56 @@
+import React, { Component } from 'react';
+import './App.css';
+
+class App extends Component {
+  render() {
+    return (
+        <div className="App">
```

Comment:
You can have an empty tag here if it's what you wanted: 
```
<>
....
</>
```
Is a valid synthax.

Reply:
"Le JS c'est du caca"  ©️React documentation on fragments with Babel.